engine/src/main/scala/cromwell/engine/workflow/lifecycle/execution/EngineJobExecutionActor.scala

```diff
@@ -337,6 +355,22 @@ class EngineJobExecutionActor(replyTo: ActorRef,
     goto(RunningJob) using data
   }
 
+  private def handleCacheInvalidatedResponse(response: CallCacheInvalidatedResponse, data: ResponsePendingData) = {
+    response match {
+      case CallCacheInvalidatedFailure(failure) => log.error(failure, "Failed to invalidate cache entry for job: {}", jobDescriptorKey)
```

Comment:
It is possible then that we end up with an invalidated cache that we retry to copy with?